package.json

```diff
@@ -44,6 +44,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "changelog": "auto-changelog -o change-output.md --unreleased-only",
```

Comment:
this does not work great, or maybe is because fabric has some old untagged commits.
This command will give you a longer list with on top the latest commit.
Is still better than what i m doing now that is typing the changelog manually.